docs/UPP_GRIB2_Table_byID.csv

```diff
@@ -0,0 +1,868 @@
+**GSD ice-friendly aerosol number concentration on model surface**,GSD_NCIFA_ON_HYBRID_LVL,PMTC,767
+Echo top height (Highest height in meters of the 18-dBZ reflectivity on a model level),GSD_ECHOTOP_ON_CLOUD_TOP,RETOP,768
+**GSD Vertically integrated liquid in kg/m^2**,GSD_VIL_ON_ENTIRE_ATMOS,VIL,769
+**GSD Vertically integrated liquid based on reflectivity factor in kg/m^2**,GSD_RADARVIL_ON_ENTIRE_ATMOS,RADARVIL,770
+**Instantaneous shortwave surface downward direct normal irradiance**,INST_SWDDNI_ON_SURFACE,VBDSF,772
+**Instantaneous shortwave surface downward diffuse irradiance**,INST_SWDDIF_ON_SURFACE,VDDSF,773
+**Model-state cloud fraction (unprocessed) on model surface**,FRACCC_ON_HYBRID_LVL,FRACCC,774
+Bucket graupel precipitation on surface,BUCKET_GRAUPEL_ON_SURFACE,FROZR,775
+Height on highest tropospheric -10C level,`HGT_ON_HGHST_TROP_-10C_LVL`,HGT,776
+Relative humidity on highest tropospheric -10C level,`RH_ON_HGHST_TROP_-10C_LVL`,RH,777
+Pressure on highest tropospheric -10C level,`PRES_ON_HGHST_TROP_-10C_LVL`,PRES,778
+Height on highest tropospheric -20C level,`HGT_ON_HGHST_TROP_-20C_LVL`,HGT,779
+Relative humidity on highest tropospheric -20C level,`RH_ON_HGHST_TROP_-20C_LVL`,RH,780
+Pressure on highest tropospheric -20C level,`PRES_ON_HGHST_TROP_-20C_LVL`,PRES,781
+**Accumulated freezing rain**,ACM_FRAIN_ON_SURFACE,FRZR,782
+Maximum u wind on 10 meter above ground,MAX_UGD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m,MAXUW,783
+Maximum v wind on 10 meter above ground,MAX_VGD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m,MAXVW,784
+Maximum reflectivity on -10C surface,`MAX_REF_ON_ISOTHERMAL_-10C`,REFD,785
+**Global Systems Division (GSD) Minimum updraft helicity 2-5km**,GSD_MIN_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_2-5km,MNUPHL,786
+**GSD Minimum updraft helicity 1-6 km**,GSD_MIN_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_1-6km,MNUPHL,787
+**GSD Maximum updraft helicity 0-2km**,GSD_MAX_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-2km,MXUPHL,788
+**GSD Minimum updraft helicity 0-2km**,GSD_MIN_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-2km,MNUPHL,789
+**GSD Maximum updraft helicity 0-3km**,GSD_MAX_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-3km,MXUPHL,790
+**GSD Minimum updraft helicity 0-3km**,GSD_MIN_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-3km,MNUPHL,791
+**GSD Maximum relative vertical vorticity 0-2 km**,GSD_MAX_REL_VORT_ON_SPEC_HGT_LVL_ABOVE_GRND_0-2km,RELV,792
+**GSD Maximum relative vertical vorticity 0-1 km**,GSD_MAX_REL_VORT_ON_SPEC_HGT_LVL_ABOVE_GRND_0-1km,RELV,793
+**GSD Maximum hail diameter in column**,GSD_MAX_COLMAX_GRAUPEL_HAIL_DIAMETER,HAIL,794
+**GSD Maximum hail diameter at k=1**,GSD_MAX_SIGMA_LVL_MAX_GRAUPEL_HAIL_DIAMETER,HAIL,795
+**Experimental cloud base pressure**,GSD_PRES1_ON_CLOUD_BASE,PRES,798
+**GSD maximum cloud fraction in (PBL + 1 km)**,TCDC_ON_BOUND_LYR,TCDC,799
+**Shelter level apparent temperature**,APTMP_ON_SPEC_HGT_LVL_ABOVE_GRND_2m,APTMP,808
+SSMIS F16 (183H  19H  19V  37H  37V  85H  85V),Not currently available for grib2,NA,818
```

Comment:
I was a little confused about this, since there seem to be SSMI and SSMIS satellites. That issue refers to SSMI, but I wasn't sure if it also covered SSMIS or what the difference was...

Reply:
The issue is for SSMI 13-20 (actually named ssmi 13-15 and ssmis 16-20) but it is all the same. After thinking more, since these aren't in post_avblflds.xml, I would actually just remove them from the table, especially since they are slated for removal from the code.